lib/src/cancelable_operation.dart

```diff
@@ -212,6 +221,15 @@ class CancelableCompleter<T> {
   }
 
   /// Cancel the completer.
+  ///
+  /// This call will be ignored if the result of the operation is already
+  /// available.
+  /// This condition may happen some time after [isComplete] is true if
```

Comment:
Which "condition"? Guessing "result of the operation is .. .available".

Maybe:

/// The result of the operation may only be available some time after
/// the completer has been completed (using [complete] or [completeError],
/// which sets [isCompleted] to true) if completed with a [Future].
/// The completer can be cancelled until the result becomes available,
/// even if [isCompleted] is true.